### PR TITLE
apt, wajig: add Polish translation

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/linux/wajig.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Uproszczone narzędzie do zarządzania pakietami dla systemów oparych na Debianie.
+> Więcej informacji: <https://wajig.togaware.com>.
+
+- Aktualizacja listy dostępnych pakietów i ich wersji:
+
+`wajig update`
+
+- Instalacja pakietu lub aktualizacja do najnowszej wersji:
+
+`wajig install {{package}}`
```
Comment (navarroaxel) [inline pages.pl/linux/wajig.md:12]:
```suggestion
`wajig install {{pakiet}}`
```

File: pages.pl/linux/wajig.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Usunięcie pakietu i jego plików konfiguracyjnych:
+
+`wajig purge {{package}}`
```
Comment (navarroaxel) [inline pages.pl/linux/wajig.md:16]: Please translate the placeholders for get this easier for non-English speakers.
```suggestion
`wajig purge {{pakiet}}`
```

File: pages.pl/linux/wajig.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`wajig versions`
+
+- Lista wersji pakietów możliwych do aktualizacji:
+
+`wajig toupgrade`
+
+- Wyświetlenie pakietów, które posiadają zaleność od podanego pakietu:
+
+`wajig dependents {{package}}`
```
Comment (navarroaxel) [inline pages.pl/linux/wajig.md:36]:
```suggestion
`wajig dependents {{pakiet}}`
```

**Comment by navarroaxel** [comment]
You don't need to keep your PR in sync with `tldr-pages:main` because these are independent pages. You should amend your commit using:

```bash
git commit --amend --author 'Your name <user@example.com>'
git push --force
```

And the name and email must match your GitHub account to pass the CLA check.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**